Integrations/UrlScan/URLScan.py

```diff
@@ -146,7 +146,7 @@ def poll(target, step, args=(), kwargs=None, timeout=None, max_tries=None, check
 
 def urlscan_submit_url():
     submission_dict = {}
-    if demisto.args().get('public') == 'public':
+    if demisto.params().get('privacy') == 'public':
```

Comment:
Seems like you disabled the argument of the command.
We should talk about it again because this breaks BC anyway.

Reply:
I agree. I'm not sure how we would both change the default and not break BC. I could change the argument do do something like:
```
if demisto.args().get('public') == 'public' and demisto.params().get('override) is not True:
   do the thing
```